packages/kit/src/core/sync/write_tsconfig.spec.js

```diff
@@ -23,7 +24,9 @@ test('Creates tsconfig path aliases from kit.alias', () => {
 		'simpleKey/*': ['../simple/value/*'],
 		key: ['../value'],
 		'key/*': ['../some/other/value/*'],
-		keyToFile: ['../path/to/file.ts']
+		keyToFile: ['../path/to/file.ts'],
+		$routes: ['./types/src/routes'],
```

Comment:
shouldn't this be `./.svelte-kit/types/src/routes`?

Reply:
The generated tsconfig is located in .svelte-kit so the types folder is in the current directory